css/style.css

```diff
@@ -23,35 +31,81 @@ h1 {
 .main {
   border: 2px solid #085311;
   height: 720px;
-  width: 370px;
+  margin: 0px 15px;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
+/*cards and content*/
 .card {
   display: flex;
   flex-direction: column;
   align-items: center;
   border: 2px solid #53d562;
   box-shadow: 0px 0px 7px 2px;
-  margin: 25px 15px 5px;
-  width: 300px;
+  margin: 25px 0px;
+  width: 84%;
   height: 200px;
 }
 
-.question {
+.card__question {
   text-decoration: underline;
   text-decoration-color: rgb(47, 200, 47);
   text-underline-offset: 2px;
 }
 
-.answer-button {
+.card__answer-button {
   background-color: #000;
   color: #08f223;
   border: 2px solid #53d562;
   padding: 2px 3px;
   font-family: "space-mono", mono;
   font-weight: 400;
   font-style: normal;
+  cursor: pointer;
+}
+
+.card__answer {
+  color: blue;
+}
+
+.card__answer--visible {
+  color: yellow;
```

Comment:
If there is a `visible` state, then line 69-71 would refer to the `not visible` state, wouldn't it? If so, why setting the color for something which is not visible?